op-batcher/Makefile

```diff
@@ -1,6 +1,12 @@
 GITCOMMIT ?= $(shell git rev-parse HEAD)
 GITDATE ?= $(shell git show -s --format='%ct')
-VERSION := v0.0.0
+# Find the latest github tag that contains this commit. If none are found, set the version string to "untagged"
+VERSION ?= $(shell tag=$$(git tag --contains $(GITCOMMIT) | grep '^op-batcher/' | sort -V | tail -n 1 | sed 's/op-batcher\///'); \
```

Comment:
I think we may be able to use `--points-at` instead of `--contains` here
https://stackoverflow.com/questions/4545370/how-to-list-all-tags-pointing-to-a-specific-commit-in-git

Reply:
Yes that seems like a better option. No extraneous historical tag info produced by `--points-at`. Will push that change